src/components/elements/dropdown-search.tsx

```diff
@@ -45,7 +67,7 @@ const DropdownSearch = ({
     onChange,
     selected_item,
     ...props
-}) => {
+}: DropdownProps) => {
     const [input_value, setInputValue] = useState('')
```

Comment:
Please check here and in all PR types for useState

Reply:
Was decided to remove whole component, because we use him...nowhere